spec/maxitest_spec.rb

```diff
@@ -296,7 +296,7 @@ def with_env(h)
     ENV.replace old
   end
 
-  def sh(command, options={})
+  def run_cmd(command, options={})
```

Comment:
makes it hard to see what the real diff is 😞

Reply:
it just overwrite a ruby keyword and I avoid to do that because some new version of ruby behave differently when we do that.

https://apidock.com/ruby/FileUtils/sh

Build green